src/test/java/guitests/KeyboardShortcutsTest.java

```diff
@@ -169,10 +169,13 @@ public void keyboardShortcutsTest() {
         // test shortcut on repo dropdown
         doubleClick(repoSelectorComboBox);
         pushKeys(SWITCH_DEFAULT_REPO);
-        waitUntilNodeAppears("#dummy/dummy_col1_1");
+        // wait for issue 9 to appear then click on it
+        // issue 9 is chosen instead of issue 10
+        // as there is a problem with finding issue 10's node due to it being the first card in the panel
+        waitUntilNodeAppears("#dummy/dummy_col1_9");
```

Comment:
'Works for 9 but not for 10' can often be traced to a short-sighted algorithm that did not cater for double-digit numbers. Sure it's not the case here?
'Doesn't work for the first one' can often be traced to a 'starts from 0 instead of 1' indexing problem.